packages/apps/plugins/plugin-sketch/src/meta.tsx

```diff
@@ -7,7 +7,7 @@ import React from 'react';
 
 import { pluginMeta } from '@dxos/app-framework';
 
-// TODO(burdon): Rename TLDRAW_PLUGIN.
+// TODO(burdon): Rename TLDRAW_PLUGIN?
```

Comment:
I think tldraw aims less to be a recognizable product and wants to be white-labelled. Most people won't know what tldraw is so we should name this literally (sketch, diagram, whiteboard, etc.) Similarly, Excalidraw is a bit more of a product and people who want it will want it because they specifically like excalidraw, so that makes more sense to re-use their name.

Reply:
ok good point.